### DISCUSSION TITLE
bw开票信息发了，已经可以抓到票信息了

### DISCUSSION THREAD

**Discussion by Nokoyoku** [mikumifa/biliTickerBuy#184] [提问专区] [answered]
![image](https://github.com/mikumifa/biliTickerBuy/assets/121370149/6c0b9824-0a21-4d39-af35-5126275c369f)
大佬看看应该没问题能抢吧

**Comment by WittF** [comment, marked as answer]
https://github.com/mikumifa/biliTickerBuy/discussions/163
